From ltsi-dev-bounces@lists.linuxfoundation.org Fri Jul  6 07:29:37 2012
From: Simon Horman <horms@verge.net.au>
Date: Fri,  6 Jul 2012 23:29:18 +0900
Subject: ARM: mach-shmobile: armadillo800eva: defconfig disable CONFIG_SYSFS_DEPRECATED
To: ltsi-dev@lists.linuxfoundation.org
Cc: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.yj@renesas.com>
Message-ID: <1341584958-17096-1-git-send-email-horms@verge.net.au>


In adition to the change proposed for upstream, LTSI requires that
CONFIG_SYSFS_DEPRECATED be removed from the defconfig in order to allow
allows use a Debian armhf usespace as well as the existing ability to use a
Debian armel userspace.

N.B: Not necessary upstream: CONFIG_SYSFS_DEPRECATED is not in the defconfig

Cc: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.yj@renesas.com>
Signed-off-by: Simon Horman <horms@verge.net.au>

---

Hi Greg,

please consider this for inclusion in LTSI.
---
 arch/arm/configs/armadillo800eva_defconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/configs/armadillo800eva_defconfig
+++ b/arch/arm/configs/armadillo800eva_defconfig
@@ -7,7 +7,6 @@ CONFIG_LOG_BUF_SHIFT=16
 # CONFIG_IPC_NS is not set
 # CONFIG_USER_NS is not set
 # CONFIG_PID_NS is not set
-CONFIG_SYSFS_DEPRECATED=y
 CONFIG_SYSFS_DEPRECATED_V2=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 CONFIG_SLAB=y
